### ISSUE TITLE
Smaller genome size than expected

### ISSUE THREAD

**Issue by a-velt** [PacificBiosciences/pbbioconda#191]
Hi,

**Operating system**
CentOS7

**Package name**
fc_run

**Conda environment**

```
> # packages in environment at /usr/local/bioinfo/src/Miniconda/Miniconda3-4.4.10/envs/pbbioconda_805fb87_env:
> #
> # Name                    Version                   Build  Channel
> _libgcc_mutex             0.1                        main
> asn1crypto                0.24.0                   py27_0
> avro                      1.8.0                    py27_0    bioconda
> avro-python2              1.9.0                      py_0    bioconda
> bam2fastx                 1.3.0                         0    bioconda
> bax2bam                   0.0.9                h707fff8_6    bioconda
> bedtools                  2.28.0               hdf88d34_0    bioconda
> blas                      1.0                         mkl
> blasr                     5.3.3                h707fff8_0    bioconda
> blasr_libcpp              5.3.3                h707fff8_0    bioconda
> boost                     1.67.0                   py27_4
> bwa                       0.7.17               hed695b0_6    bioconda
> bzip2                     1.0.8                h7b6447c_0
> ca-certificates           2019.5.15                     1
> certifi                   2019.6.16                py27_1
> cffi                      1.12.3           py27h2e261b9_0
> chardet                   3.0.4                 py27_1003
> cryptography              2.7              py27h1ba5d50_0
> curl                      7.65.3               hbc83047_0
> cython                    0.29.13          py27he6710b0_0
> decorator                 4.4.0                    py27_1
> enum34                    1.1.6                    py27_1
> future                    0.17.1                   py27_0
> genomicconsensus          2.3.2                    py27_1    bioconda
> h5py                      2.9.0            py27h7918eee_0
> hdf5                      1.10.4               hb1b8bf9_0
> htslib                    1.9                  h4da6232_3    bioconda
> icu                       58.2                 h9c2bf20_1
> idna                      2.8                      py27_0
> intel-openmp              2019.4                      243
> ipaddress                 1.0.22                   py27_0
> iso8601                   0.1.12                   py27_1
> isoseq3                   3.1.2                         0    bioconda
> krb5                      1.16.1               h173b8e3_7
> libboost                  1.67.0               h46d08c1_4
> libcurl                   7.65.3               h20c2e04_0
> libdeflate                1.0                  h14c3975_1    bioconda
> libedit                   3.1.20181209         hc058e9b_0
> libffi                    3.2.1                hd88cf55_4
> libgcc                    7.2.0                h69d50b8_2
> libgcc-ng                 9.1.0                hdf63c60_0
> libgfortran-ng            7.3.0                hdf63c60_0
> libssh2                   1.8.2                h1ba5d50_0
> libstdcxx-ng              9.1.0                hdf63c60_0
> lima                      1.9.0                         0    bioconda
> linecache2                1.0.0                    py27_0
> minimap2                  2.17                 h84994c4_0    bioconda
> minorseq                  1.11.0                        1    bioconda
> mkl                       2019.4                      243
> mkl-service               2.0.2            py27h7b6447c_0
> mkl_fft                   1.0.14           py27ha843d7b_0
> mkl_random                1.0.2            py27hd81dba3_0
> mummer4                   4.0.0beta2      pl526hf484d3e_4    bioconda
> ncurses                   6.1                  he6710b0_1
> networkx                  2.2                      py27_1
> nim-falcon                0.0.1                h2f8a211_0    bioconda
> numpy                     1.11.3          py27h7e9f1db_12
> numpy-base                1.11.3          py27hde5b4d6_12
> openssl                   1.1.1c               h7b6447c_1
> pb-assembly               0.0.2                    py27_0    bioconda
> pb-dazzler                0.0.1                h516909a_0    bioconda
> pb-falcon                 0.2.7            py27hc0aa232_1    bioconda
> pbalign                   0.3.1                    py27_0    bioconda
> pbbam                     0.23.0               h88345a9_0    bioconda
> pbccs                     4.0.0                         0    bioconda
> pbcommand                 1.1.1            py27h24bf2e0_1    bioconda
> pbcopper                  0.4.2                h3f2a3d2_1    bioconda
> pbcore                    1.5.1                    py27_1    bioconda
> pbcoretools               0.2.4                    py27_1    bioconda
> pbgcpp                    0.0.1                         0    bioconda
> pblaa                     2.4.2                         1    bioconda
> pbmm2                     1.0.0                ha888412_0    bioconda
> pbsv                      2.2.1                         0    bioconda
> pcre                      8.43                 he6710b0_0
> perl                      5.26.2               h14c3975_0
> pip                       19.2.2                   py27_0
> py-boost                  1.67.0           py27h04863e7_4
> pycparser                 2.19                     py27_0
> pyopenssl                 19.0.0                   py27_0
> pysam                     0.15.3           py27hda2845c_1    bioconda
> pysocks                   1.7.0                    py27_0
> python                    2.7.16               h8b3fad2_5
> python-consensuscore      1.1.1            py27h02d93b8_1    bioconda
> python-consensuscore2     3.1.0                    py27_1    bioconda
> python-edlib              1.3.4            py27hc9558a2_0    bioconda
> python-intervaltree       3.0.2                      py_0    bioconda
> python-msgpack            0.6.1            py27h6bb024c_0    bioconda
> python-sortedcontainers   2.1.0                      py_0    bioconda
> pytz                      2019.2                     py_0
> racon                     1.4.3                he513fc3_0    bioconda
> readline                  7.0                  h7b6447c_5
> recalladapters            7.1.0                         0    bioconda
> requests                  2.22.0                   py27_0
> samtools                  1.9                 h8571acd_11    bioconda
> setuptools                41.0.1                   py27_0
> six                       1.12.0                   py27_0
> sqlite                    3.29.0               h7b6447c_0
> tk                        8.6.8                hbc83047_0
> traceback2                1.4.0                    py27_0
> unittest2                 1.1.0                    py27_0
> urllib3                   1.24.2                   py27_0
> wheel                     0.33.4                   py27_0
> xz                        5.2.4                h14c3975_4
> zlib                      1.2.11               h7b6447c_3
> 
```

**Describe the bug**
After the assembly step with fc_run, I get a genome size of 297 Mb, this size does not change after the 3-unzip and 4-polish steps. The problem is that I know my genome size is about 450Mb/500Mb. This is not a bug, but I don't know what settings to change to fix this problem.

**Error message**
No error message.

**To Reproduce**
Here my configuration : 
```
[General]
# list of files of the initial bas.h5 files
input_fofn = /FALCON/data/input.fofn

input_type = raw
#input_type = preads

# The length cutoff used for seed reads used for initial mapping
length_cutoff = 5000
genome_size = 500000000

# The length cutoff used for seed reads usef for pre-assembly
length_cutoff_pr = 5000

pa_daligner_option =  -e.70 -l2500 -s100 -w8 -h1250 -k18
ovlp_daligner_option = -h 60 -e.96 -l1500 -s100 -k24 -h1250

pa_HPCdaligner_option =  -v -B128 -M10
ovlp_HPCdaligner_option = -v -B128 -M10

pa_DBsplit_option = -a -x500 -s200
ovlp_DBsplit_option = -x500 -s200

falcon_sense_option = --output_multi --min_idt 0.70 --min_cov 4 --max_n_read 400 --n_core 15

overlap_filtering_setting = --max_diff 120 --max_cov 120 --min_cov 4 --n_core 15 --bestn 10

[job.defaults]
job_type = string
pwatcher_type = blocking

#use_tmpdir = /scratch
#submit = bash -C ${CMD} >| ${STDOUT_FILE} 2>| ${STDERR_FILE}
#job_queue = bash -C ${CMD}
# By dropping STD*_FILE, we see all output on the console.
# That helps debugging in TravisCI/Bamboo.

submit = sbatch --wait -p workq -J ${JOB_NAME} -o ${JOB_STDOUT} -e ${JOB_STDERR} --mem-per-cpu=${MB}M --cpus-per-task=${NPROC} ${JOB_SCRIPT}

MB = 200000
NPROC=2
#njobs=1
[job.step.da]
[job.step.pda]
[job.step.la]
NPROC=2
[job.step.pla]
NPROC=2
[job.step.cns]
#njobs=1
NPROC=2
[job.step.asm]
NPROC=2
```

**Expected behavior**
I'm waiting for a genome size of about 450Mb/500Mb.

Best,
Amandine

**Comment by a-velt** [comment]
I tried to increase the overlap_filtering_setting with following settings : 

`overlap_filtering_setting = --max_diff 500 --max_cov 500 --min_cov 4 --n_core 15
`
But the result is worse, my genome size is about 236Mb.

What settings can I play on?

Here my DBstats on rawreads : 

> Statistics for all wells of length 500 bases or more
> 
>       4,714,633 reads        out of       5,002,392  ( 94.2%)
>  58,782,288,291 base pairs   out of  58,865,232,055  ( 99.9%)
> 
>          12,468 average read length
>          11,449 standard deviation
> 
>   Base composition: 0.306(A) 0.195(C) 0.194(G) 0.305(T)

> Here my DBstats on preads : 
> 
> Statistics for all reads of length 500 bases or more
> 
>         280,118 reads        out of         280,118  (100.0%)
>   5,806,052,696 base pairs   out of   5,806,052,696  (100.0%)
> 
>          20,727 average read length
>           9,428 standard deviation
> 
>   Base composition: 0.315(A) 0.184(C) 0.184(G) 0.317(T)

I have a cover of my genome over 100X with PacBio raw reads (my genome size is about 500Mb). And then only 11X with preads. Is it normal that the number of bases is so reduced?

**Comment by armintoepfer** [comment]
Hi Amandine,

in this issue tracker, we as developers try to help out if something crashes.
But we can rarely cover specific data or project problems, because might be very specific to your library that makes it near impossible for us to troubleshoot. Maybe one of the assembly developers has an idea or gut feeling, but please don't expect an answer too soon, because this is issue tracker is not covered by service contracts.

Armin

**Comment by a-velt** [comment]
Ok, I understand, thank you.
Another piece of information that may have helped, I had already assembled genomes of my species with Pacbio's P6-C4 RSII technology and FALCON, with no problem of size and with a good quality assembly.
Here, the data I used are from the Sequel system, so different chemistry.
But I used same FALCON options for all the assemblies. Maybe some options have to change for the data from Sequel system ? 
I hope someone will have an idea. In the meantime, thank you for your answer.
Amandine
